project1/css/index.css

```diff
@@ -62,3 +62,103 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+
+.container {
+	width: 1100px;
+	margin: auto;
+}
+
+header {
+	display: inline-block;
+	margin-top: 20px;
+	margin-bottom: 30px;
+	width: 100%;
+}
+
+nav {
+	display: inline-block;
+	margin-left: 80px;
+	margin-right: 100px;
+}
+
+nav ul li {
+	display: inline-block;
+	margin: 25px;
+}
+
```

Comment:
The image shows that the links should be grey.
![links](https://user-images.githubusercontent.com/12449668/37050931-6a501480-2132-11e8-8d94-8107c081df1c.png)